jdaviz/configs/specviz2d/helper.py

```diff
@@ -164,9 +166,14 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
                     spext.export_extract_spectrum(add_data=True)
                 except Exception:
                     msg = SnackbarMessage(
-                        "Automatic spectrum extraction failed.  See the spectral extraction"
+                        "Automatic spectrum extraction failed. See the spectral extraction"
                         " plugin to perform a custom extraction",
-                        color='error', sender=self, timeout=10000)
+                        color='warning', sender=self, timeout=10000)
+                    # Falls back to pre-specreduce way of doing things.
+                    self.app.load_data(spectrum_2d,
+                                       parser_reference="spec2d-1d-parser",
+                                       data_label=spectrum_1d_label,
+                                       show_in_viewer=show_in_viewer)
```

Comment:
Interesting idea to have a fallback (although if we know a case that doesn't work, we should fix that case)... I just worry since it's a very different treatment and the settings in the extraction plugin won't match, that this could be very confusing to the user.  Maybe instead we should have the extraction plugin itself know to fallback to a boxcar with the full-width of the input image?

Reply:
As discussed offline, feel free to open another PR with alternate solution.

I am not familiar with specreduce and it keeps crashing it in some background extraction step that is hidden from me here.

Reply:
FWIW, I copy-pasted this code from an older version of Specviz2d. So, this used to work (and still works) for the affected data, but no longer works with the new way of things, so in a sense, this is a regression.

Reply:
this fallback should be able to be removed with #1633